css_tasks/task11/scss/_footer.scss

```diff
@@ -0,0 +1,220 @@
+@import "variables";
+@import "mixins";
+
+.footer {
+  background: $decorationBackground;
+}
+.footer__container {
+  position: relative;
+  display: flex;
+  padding-top: $standardTopPadding;
+  padding-bottom: 4.3125rem;
+}
+.footer__logo {
+  font-family: $secondaryFont;
+  font-size: calc(12px + (46 - 12) * ((100vw - 300px) / (1920 - 300)));
```

Comment:
i understand that it is for responsive but it looks strange and it better to use mixin for something like this, better with some comment what happening there